### ISSUE TITLE
ContainerSSH Metrics are not Prometheus Compatible

### ISSUE THREAD

**Issue by ChristopherJMiller** [ContainerSSH/ContainerSSH#537]
## Describe the bug

Attempting to scrape the metrics endpoint into prometheus fails with the error:

```
unit not a suffix of metric "containerssh_config_server_failures"
```

In determining why I was receiving this error, I found that according to the [metrics and labels](https://prometheus.io/docs/practices/naming/#metric-names) documentation for prometheus, all metrics must end with a plural suffix of it's unit (which looks to be set by the comment to be "requests").

## To Reproduce

1. Enable metrics
2. Set as scrape target for Prometheus

## Expected behavior

Scrapes successfully

## Version

0.4.1

**Comment by ChristopherJMiller** [comment]
I see this was fixed in libcontainerssh here https://github.com/ContainerSSH/libcontainerssh/pull/93

Is there an ETA on when a new version of ContainerSSH will be released with this fix?

**Comment by tsipinakis** [comment]
Closing this as it was fixed with the release of 0.5.
